docs/maps/maps-aggregations.asciidoc

```diff
@@ -68,9 +68,9 @@ To enable a blended layer that dynamically shows clusters or documents:
 
 [role="xpack"]
 [[maps-top-hits-aggregation]]
-=== Top hits per entity
+=== Display the most recent documents
```

Comment:
This title is not completely accurate. The top hits per entity is determined by the sort field and sort direction. Users can change either so that the top hits are not the most recent documents - but the most "relevent documents". Is this a concern that the title is too limiting?

Reply:
Good point.

I had the title as Display the most recent and relevant documents, but that was too long. Most relevant is better. The concern is the title is ambiguous. The goal was to make it more use-case driven.

@nreese Propose to change to: **Display the most relevant documents**

Reply:
I think the term "entity" needs to be in there somewhere. The big distinction with the top hits layer is that it grabs the top hits per entity instead of just grabbing the first 10000 matching documents.